hmpps-kotlin-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/kotlin/customize/ResourceServerConfigurationCustomizerDsl.kt

```diff
@@ -0,0 +1,50 @@
+package uk.gov.justice.hmpps.kotlin.customize
+
+@DslMarker
+annotation class ResourceServerConfigurationCustomizerDslMarker
```

Comment:
Decided to prefer a customization DSL over property based customization. This is mainly because the syntax is nicer and gives us autocompletion, type checking and other niceties of using a real language.

Should we require customizations that differ between environments then we may also add some configuration properties, but currently no need to.